### PR TITLE
fix(firebase_ui_auth): Fixed icon only oauth variant such that the oauth buttons are displayed in a row without text

### PR DESCRIPTION
**Pull request by bclarksoftware**
## Description

*I noticed this issue when attempting to utilize the SignInScreen on my personal app. When passing in the oauthButtonVariant of icon, the oauth buttons would return in a row including their text causing an overflow. The ideal behavior is to return the buttons in a row without their text. In order to fix this issue, we need to fully pass in the users variable down through each widget; in this case, we only missed passing the parameter in the LoginView.*

## Related Issues

*I did not find any related issue for this bug.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

There were a few unrelated failures when running the 'melos run analyze' command. I did not add any additional unit tests as I've never written a single test using dart before. Please let me know if we really need the tests for this parameter passing; I'd be willing to quickly add it if necessary.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore_interop.dart [hunk 1]
```diff
@@ -480,7 +479,8 @@ abstract class TimestampJsImpl {
 @anonymous
 @JS()
 abstract class FirestoreError {
-  external String /*|'cancelled'|'unknown'|'invalid-argument'|'deadline-exceeded'|'not-found'|'already-exists'|'permission-denied'|'resource-exhausted'|'failed-precondition'|'aborted'|'out-of-range'|'unimplemented'|'internal'|'unavailable'|'data-loss'|'unauthenticated'*/ get code;
+  external String /*|'cancelled'|'unknown'|'invalid-argument'|'deadline-exceeded'|'not-found'|'already-exists'|'permission-denied'|'resource-exhausted'|'failed-precondition'|'aborted'|'out-of-range'|'unimplemented'|'internal'|'unavailable'|'data-loss'|'unauthenticated'*/
+      get code;
 
   external set code(
       /*|'cancelled'|'unknown'|'invalid-argument'|'deadline-exceeded'|'not-found'|'already-exists'|'permission-denied'|'resource-exhausted'|'failed-precondition'|'aborted'|'out-of-range'|'unimplemented'|'internal'|'unavailable'|'data-loss'|'unauthenticated'*/
```
Comment (lesnitsky) [inline packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore_interop.dart:486]: changes in this file are not related to the actual issue this PR is intended to fix, so please revert these

**Review by lesnitsky — CHANGES_REQUESTED** [review]
there are changes unrelated to the PR's purpose

File: packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore_interop.dart (hunk 1 above)
Comment (bclarksoftware) [reply packages/cloud_firestore/cloud_firestore_web/lib/src/interop/firestore_interop.dart]: Sorry about the unintended changes. Those probably snuck in when I ran the formatter perhaps? Anyways, I updated the branch to only include the intended changes. Please let me know if there is anything else I can do. Thank you for the fast review!

**Comment by bclarksoftware** [comment]
@lesnitsky - Is there anything else needed for your approval?
